database/migrations/2017_07_22_174441_add_role_to_users.php

```diff
@@ -0,0 +1,23 @@
+<?php
```

Comment:
Recomendaria crear una tabla para roles y una tabla pivot para hacer el relationship entre roles y usuarios. Con eso es sencillo aplicar la parte de Authorizartion que da laravel:
[Authorization](https://laravel.com/docs/5.4/authorization#via-the-user-model)

Otra opcion es usar este paquete, que trae casi todo listo:
[Zizaco/entrust](https://github.com/Zizaco/entrust)